src/shared/components/common/paginator-cursor.tsx

```diff
@@ -0,0 +1,46 @@
+import { Component, linkEvent } from "inferno";
+import { I18NextService } from "../../services";
+import { PaginationCursor } from "lemmy-js-client";
+
+interface PaginatorCursorProps {
+  prevPage?: PaginationCursor;
+  nextPage?: PaginationCursor;
+  onNext(val: PaginationCursor): any;
```

Comment:
Do we use the return value for this? If not, we could make this return `void` as well.